content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -79,12 +79,10 @@ An example of an IPv6 CIDR: `fdXY:IJKL:MNOP:15::/64` (this shows the format but
 address - see [RFC 4193](https://tools.ietf.org/html/rfc4193))
 {{< /note >}}
 
```

Comment:
Better to put a heading here.
```suggestion

### Custom node IP address settings

```
Even better though: move this inside the `kubelet` item earlier in the section, and reword accordingly. That's really what I'd like.

Reply:
> move this inside the `kubelet` item earlier in the section

oh, right, good catch